Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -3603,6 +3603,77 @@ bool CStaticFunctionDefinitions::SetElementCollidableWith ( CClientEntity & Enti
 }
 
 
+bool CStaticFunctionDefinitions::SetElementCollidableWithType ( CClientEntity & Entity, const char* szTypeName, bool bCanCollide, bool bOnlyWithCreated )
+{
+    switch ( Entity.GetType () )

```

Comment:
You have a few nested code blocks here, but you could do **return false;** in the **default** case and do **break** for the supported entity types and continue after the switch.